tsl/src/compression/compression.h

```diff
@@ -391,6 +391,13 @@ consumeCompressedData(StringInfo si, int bytes)
 	return result;
 }
 
+/* build scankeys for segmentby columns */
+#define SEGMENTBY_KEYS (1 << 1)
+/* build scankeys for orderby columns */
+#define ORDERBY_KEYS (1 << 2)
```

Comment:
Do you need this define? It seems it is never used in the code.

Reply:
Removed ORDERBY_KEYS as it is not used anywhere